typespec-tests/src/test/java/com/cadl/flatten/generated/FlattenOpSendLongTests.java

```diff
@@ -17,7 +17,7 @@ public final class FlattenOpSendLongTests extends FlattenClientTestBase {
     public void testFlattenOpSendLongTests() {
         // method invocation
         flattenClient.sendLong(
-            new SendLongOptions("myRequiredId", "myRequiredInput", 11, "title", SendLongRequestStatus.NOT_STARTED)
+            new SendLongOptions("myRequiredId", "myInput", 0, null, "title", SendLongRequestStatus.NOT_STARTED)
```

Comment:
0 is absence of required primitive.
null is absence of required model.

Reply:
Do we have document somewhere to list different type's default value?

Reply:
I wasn't able to find any. 

In Javadoc, for general case, it's in `IType`:https://github.com/Azure/autorest.java/blob/403ffcb32add0d2b9457bb86494ca68ae15605c5/javagen/src/main/java/com/azure/autorest/model/clientmodel/IType.java#L81-L90

For other scenarios, you'll need to check each sub type's definition, e.g. https://github.com/Azure/autorest.java/blob/403ffcb32add0d2b9457bb86494ca68ae15605c5/javagen/src/main/java/com/azure/autorest/model/clientmodel/ClassType.java#L197

Reply:
here https://github.com/Azure/autorest.java/blob/403ffcb32add0d2b9457bb86494ca68ae15605c5/javagen/src/main/java/com/azure/autorest/model/clientmodel/PrimitiveType.java#L36